ps_categorytree.php

```diff
@@ -161,7 +161,7 @@ private function getCategories($category)
 			AND c.id_category IN (
 				SELECT id_category
 				FROM `' . _DB_PREFIX_ . 'category_group`
-				WHERE `id_group` IN (' . pSQL(implode(', ', Customer::getGroupsStatic((int) $this->context->customer->id))) . ')
+				WHERE `id_group` IN (' . implode(', ', array_map('intval', Customer::getGroupsStatic((int) $this->context->customer->id))) . ')
```

Comment:
It seems that the whole usage of pSQL is useless here, this is not a value took from a user, and [it's already casted to `int` in `Customer::getGroupsStatic()`](https://github.com/PrestaShop/PrestaShop/blob/1.7.8.x/classes/Customer.php#L1071)

Reply:
Thanks @atomiix I've deleted the psql